workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -160,5 +163,26 @@ private void DrawCodeGenerationOptions()
 
             EditorGUIUtility.labelWidth = previousWidth;
         }
+
+        private void DrawCustomSnapshotDir()
+        {
+            GUILayout.Label(CustomSnapshotPathLabel, EditorStyles.boldLabel);
+            GUILayout.Space(EditorGUIUtility.standardVerticalSpacing);
+
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                EditorGUI.BeginDisabledGroup(true);
+                    EditorGUILayout.TextField(CustomSnapshotPathLabel, toolsConfig.CustomSnapshotPath);
+                EditorGUI.EndDisabledGroup();
+                if (GUILayout.Button("Open", EditorStyles.miniButtonRight, GUILayout.ExpandWidth(false)))
+                {
+                    string path = EditorUtility.OpenFilePanel("Select snapshot", "", "snapshot");
```

Comment:
This should open in the directory of the currently selected/default snapshot

Reply:
The field value is being loaded from the `GdkToolsConfiguration` class, which have a default value for the snapshot path (`snapshots/default.snapshot`)

Reply:
Removed the custom path from the `GdkToolsConfiguration` class and moved to a PlayerPref key